woocommerce/wc-admin
# Move settings logic out of Loader class

Fixes #8416 

This renames our current internal `Settings.php` file to `SettingsNavigationFeature.php` as it only pertains to the settings feature. I also created a new `Settings.php` class to handle all the settings logic that was still in the Loader.php file.

### Detailed test instructions:

- Load **WooCommerce > Home** and open your console, check if `window.wcSettings.admin` contains all the admin settings
- Enable the Settings feature in `includes/feature-config.php` and go to **WooCommerce > Settings > Advanced > Features** to enable it
- Check if this still works (do notice that only Settings general page works under this feature, every thing else doesn't, which is why we have it disabled by default).


No changelog necessary as this is related to the original Loader refactor PR.
